src/v2/Apps/Articles/Components/NewsIndexArticles.tsx

```diff
@@ -42,10 +43,21 @@ export const NewsIndexArticles: FC<NewsIndexArticlesProps> = ({
       <Join separator={<Separator my={4} />}>
         {articles.map((article, i) => {
           return (
-            <ArticleBodyFragmentContainer
-              key={article.internalID}
-              article={article}
-            />
+            <Fragment key={article.internalID}>
+              <ArticleBodyFragmentContainer article={article} />
+
+              {/* Insert an ad after every 6th, article; beginning with 3rd */}
+              {(i - 2) % 6 === 0 && (
+                <ArticleAd
+                  size="970x250"
+                  unit={(() => {
+                    if (i === 2) return "Desktop_Leaderboard1"
+                    if (i === 8) return "Desktop_Leaderboard2"
```

Comment:
ditto with these numbers. The comment and code makes sense, and I don't know if it can be more straightforward, but variable names (might) help